openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
Regarding the configurability of it, I shied away from that initially because it seemed like it would be harder to track/update things cleanly as we add more resolutions and new templates. I guess I'll add it in, and just lace it with caveats of "if you customize this, you might have to manually rebuild CourseOverviews one day"?